.ccache/ccache.conf

```diff
@@ -0,0 +1,32 @@
+# For complete documentation, see https://ccache.dev/manual/4.6.1.html#_configuration
+
+# This option specifies the maximum size of the cache. Use 0 for no limit. The default value is 5G.
+# Available suffixes: k, M, G, T (decimal) and Ki, Mi, Gi, Ti (binary). The default suffix is G
+max_size = 10G
+
+# This option specifies where ccache will keep its cached compiler outputs.
+cache_dir = /mnt/ccache/
+
+#If true, ccache will compress data it puts in the cache. However, this option has no effect on how files are
+#retrieved from the cache; compressed and uncompressed results will still be usable regardless of this option.
+# The default is true.
+compression = true
+
+# This option determines the level at which ccache will compress object files using the real-time compression
+# algorithm Zstandard. It only has an effect if compression is enabled (which it is by default). In restricted space environments,
+# it is recommended to set the value of this parameter to 19 (or via environment parameter CCACHE_COMPRESSLEVEL)
+compression_level = 1
+
+# When true, ccache will just call the real compiler, bypassing the cache completely. The default is false.
+disable = false
+
+# If set to a file path, ccache will write information on what it is doing to the specified file.
+# This is useful for tracking down problems.
+log_file = /mnt/ccache/ccache.log
+
+# If set to a file path, ccache will write statistics counter updates to the specified file. This is useful
+# for getting statistics for individual builds. To show a summary of the current stats log, use ccache --show-log-stats.
+stats_log = /mnt/ccache/ccache_stats.log
+
+# If true, ccache will not discard the comments before hashing preprocessor output.
+keep_comments_cpp = true
```

Comment:
Shouldn't this be set to false?

Reply:
The default is false, my idea was that ccache processor shouldn't touch the file content at all, just hash them. Anyway, I don't think it matters much.